src/vivarium_cluster_tools/psimulate/branches.py

```diff
@@ -34,13 +34,17 @@ def from_branch_configuration(
         branch_configuration_file
             Absolute path to the branch configuration file.
         """
-        branches, input_draw_count, random_seed_count = load_branch_configuration(
-            branch_configuration_file
-        )
+        (
+            branches,
+            input_draw_count,
+            random_seed_count,
+            input_draws,
+        ) = load_branch_configuration(branch_configuration_file)
         keyspace = calculate_keyspace(branches)
-        keyspace["input_draw"] = calculate_input_draws(input_draw_count)
+        keyspace["input_draw"] = (
+            input_draws if input_draws else calculate_input_draws(input_draw_count)
```

Comment:
Nit: I think it might be cleaner to pass `input_draws` to `calculate_input_draws` as well as `input_draw_count` to `calculate_input_draws` and do this if logic inside that method.

Reply:
I can't cleanly do this without adding a new parameter (i.e., `existing_draws` can't be double-purposed) and there's another caller `add_draws`), and I think this spot is the only place we'll need this logic. I'm going to leave as-is, unless there's objection.